resource_registry.gemspec

```diff
@@ -31,18 +31,25 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.5.1'
 
-  spec.add_dependency 'dry-validation',           '~> 1.2'
-  spec.add_dependency 'dry-struct',               '~> 1.0'
-  spec.add_dependency 'dry-types',                '~> 1.0'
-  spec.add_dependency 'dry-configurable',         '~> 0.12'
-  spec.add_dependency 'dry-monads',               '~> 1.3'
-  spec.add_dependency 'dry-matcher',              '~> 0.7'
-  spec.add_dependency 'dry-container',            '~> 0.9'
+  spec.add_dependency 'dry-validation'
```

Comment:
@TreyE Is there a specific reason for removing the version specification for the dry gem versions?

Reply:
Not really, just wanted to free up our constraints to avoid version fighting as we moved into newer versions of Ruby.  I can put the constraints back if that led to quicker resolution.

Reply:
Yes please put the constraints back if it is okay.